internal/configuration/env_config.go

```diff
@@ -23,6 +23,7 @@ const (
 	defaultDebugMode                 = "false"
 	defaultHTTPSEnabled              = "false"
 	defaultGatewayTimeout            = "25s"
+	defaultAllowedOrigins            = "https://*openshift.io, https://localhost:*"
```

Comment:
should this be `*.openshift.io` instead? ?

Reply:
This will cover xyz.openshift.io and openshift.io both. Otherwise I will have to put them separately. Like
```
defaultAllowedOrigins            = "https://openshift.io, https://*.openshift.io, https://localhost:*"
```